src/dowel/logger.py

```diff
@@ -5,17 +5,18 @@
 
 The logger has 4 major steps:
 
-    1. Inputs, such as a simple string or something more complicated like
-    TabularInput, are passed to the log() method of an instantiated Logger.
+  1. Inputs, such as a simple string or something more complicated like
```

Comment:
please make sure this giant commend renders nicely in sphinx. Here's what it looks like now:
https://dowel.readthedocs.io/en/latest/_apidoc/dowel.html#module-dowel.logger

Perhaps you can actually move this content (+example) to the title page of the documentation. If you do that, it will probably render fine (markdown is supported for pages but not docstrings).

Anyway, to render the docs just do
```
cd docs
make html
xdg-open _build/html/index.html  # if on macOS, use open instead
```